extension/src/config/init/ensure-payment-custom-type.js

```diff
@@ -1,18 +1,14 @@
-const paymentCustomType = require('../../../resources/payment-custom-types.json')
+const paymentCustomType = require('../../../resources/web-components-payment-type')
 
 const utils = require('../../utils')
 
 const logger = utils.getLogger()
 
 async function ensurePaymentCustomType (ctpClient) {
-  try {
-    const { body } = await ctpClient.fetch(ctpClient.builder.types.where(`key="${paymentCustomType.key}"`))
-    if (body.results.length === 0) {
-      await ctpClient.create(ctpClient.builder.types, paymentCustomType)
-      logger.info('Successfully created payment custom type')
-    }
-  } catch (e) {
-    logger.error(e, 'Error when creating payment custom type, skipping...')
+  const { body } = await ctpClient.fetch(ctpClient.builder.types.where(`key="${paymentCustomType.key}"`))
+  if (body.results.length === 0) {
+    await ctpClient.create(ctpClient.builder.types, paymentCustomType)
+    logger.info(`Successfully created a payment type (key=${paymentCustomType.key})`)
```

Comment:
It seems you don't want to change messages as requested in reviews. Anyways it is also ok for me if you want to keep it.

Reply:
Yeah, because it's the same :) It's just different wording. And I think it's important to know that sth is successful or not at the first glance, so I put this word first.